RileyLinkBLEKit/RileyLinkBLEDevice.h

```diff
@@ -90,7 +90,7 @@ typedef NS_ENUM(NSUInteger, SubgRfspyVersionState) {
 
 - (void) connectionStateDidChange:(nullable NSError *)error;
 
-- (void) runSession:(void (^ _Nonnull)(RileyLinkCmdSession* _Nonnull))proc;
+- (void) runSession:(nonnull NSString*)name proc:(void (^ _Nonnull)(RileyLinkCmdSession* _Nonnull))proc;
```

Comment:
well at least the Swift portions have to get upgraded next week anyhow... I can do some renaming then!